src/plugins/controls/public/options_list/components/options_list_popover.test.tsx

```diff
@@ -58,17 +57,6 @@ describe('Options list popover', () => {
     showOnlySelectedButton.simulate('click');
   };
 
-  test('available options list width responds to container size', async () => {
```

Comment:
The EUI `panelMinWidth` applies to the **portal** that is created for the popover, and not to the popover itself; there is no way to get access to this portal from within the popover component, so this test is no longer relevant.